custom/icds_reports/utils/aggregation_helpers/distributed/agg_dashboard_activity.py

```diff
@@ -149,7 +151,8 @@ def get_last_agg_date(self):
                   filter(date__lt=self.date.strftime("%Y-%m-%d"), date__gt=seven_days_back).
                   aggregate(Max('date')))
 
-        return result.get('date__max',  date(1970, 1, 1))  # return the oldest date in default case
+        last_agg_date = result.get('date__max')
```

Comment:
result can contain date__max key with None value, So `result.get('date__max',  date(1970, 1, 1))` was still be `None`